lib/lumberjack/server.rb

```diff
@@ -56,7 +67,7 @@ def run(&block)
 
     def accept(&block)
       begin
-        fd = @ssl_server.accept
+        fd = @server.accept
       rescue EOFError, OpenSSL::SSL::SSLError, IOError
         # ssl handshake or other accept-related failure.
         # TODO(sissel): Make it possible to log this.
```

Comment:
I would expect tcp server errors being a subset of ssl server errors. That's why I didn't add distinguish between these too.